### ISSUE TITLE
Suggestion: Allow configuration for headless:true in Playwright

### ISSUE THREAD

**Issue by khkimball** [sandstorm/uptime-robot-to-kuma-helper#1]
Currently, headless is set to false in Playwrite, which prevents running the script on a headless server.

Editing index.ts and setting headless:true allows the script without a GUI.

**Comment by hax1** [comment]
yep, that and adding to ignore ssl certs on receiving (uptimekuma side), I created a fork where that is changed - hope that's ok.

https://github.com/hax1/uptime-robot-to-kuma-helper
